### PR TITLE
Add inline_tag fixer

### PR DESCRIPTION
**Pull request by mlocati**
What about adding a fixer to format these cases?
`<?php /* anything except a new line */ ?>`
`<?= /* anything except a new line  */ ?>`

Examples:

---

immagine

---

immagine

---

immagine

---

immagine

---

immagine

---

immagine

---

immagine

---

immagine

### CODE REVIEW

File: doc/rules/php_tag/inline_tag.rst [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+Default value: ``'one'``
+
+``semicolon``
+~~~~~~~~~~~~~
+
+Whether there should be a semi-colon at the end.
+
+Allowed types: ``bool``, ``null``
+
+Default value: ``false``
```
Comment (mvorisek) [inline doc/rules/php_tag/inline_tag.rst:35]: I would strongly prefer `true` as a default, code is more readable then and more proof to errors.

Comment (mlocati) [reply doc/rules/php_tag/inline_tag.rst:35]: Do you find more readable

```php
<?= $a; ?>
```

than

```php
<?= $a ?>
```

?

Comment (mlocati) [reply doc/rules/php_tag/inline_tag.rst:35]: @mvorisek also, why "more proof to errors"?

Comment (mvorisek) [reply doc/rules/php_tag/inline_tag.rst:35]: https://3v4l.org/5JfNF more readable when there is more than one statement

Comment (mlocati) [reply doc/rules/php_tag/inline_tag.rst:35]: IMHO semicolons in inline tags are very rare. BTW if changing this value will speedup the approval of this fixer I'd be more than happy to change jt

Comment (keradus) [reply doc/rules/php_tag/inline_tag.rst:35]: I would keep default without semicolons. To be honest, not sure whether we actually need it configurable

Comment (mlocati) [reply doc/rules/php_tag/inline_tag.rst:35]: > To be honest, not sure whether we actually need it configurable

Someone may prefer to never have trailing semicolons (like me), someone else may prefer to always have them (like @mvorisek)

**Comment by VBelozyorov** [comment]
@php-cs-fixer can I somehow help to speed up including this into one of upcoming releases?

**Comment by mlocati** [comment]
Any reason to keep this PR on hold?

**Comment by mlocati** [comment]
Is there something I can do to have this fixer accepted or refused?

**Comment by keradus** [comment]
Hi @mlocati !
Sorry for late answer here. I believe this is caused by lack of traffic in this PR. Not that many people raised need for this feature.

Is this part of a standard or used by a bigger (F)OSS community? For us that is a condition to add a rule to this project. (Otherwise, you can create a custom fixer and use that in your projects. Thanks for understanding.)

To me, it looks like only use case for this rule (correct me if I don't see other use case you have in mind!) is when mixing PHP with HTML or using raw PHP for templating. For both cases, I strongly encourage to not do it and use template system like twig or mustache. With that in mind, I would also be personally 👎  for fixer trying to make such language usage more readable, as it's encouraging ppl to use inline tags in first place (as "there are even fixers for it to make it pretty! it must be a good idea to do it!")

**Comment by mlocati** [comment]
@keradus I wrote this fixer as I wanted to use it for [ComcreteCMS](https://github.com/concrete5/concrete5/). But please remark that ConcreteCMS is mainly driven by PortlandLabs, and I'm a simple community member of it.
Of course I can implement this fixer inside the ConcreteCMS project, but since it's very helpful for having a common syntax in mixed PHP+HTML files I thought it may be useful for others too (and about that: anyone is using the template engine they prefer, I don't think anyone should force people to use a specific tool...).

**Comment by keradus** [comment]
After consideration of this request, I decided to *not* merge it.

My vision is that PHP CS Fixer should focus on monolithic files and, besides from previously shared discouragement to mix PHP with HTML in a single file, if we would start claiming any support for templates will open the pandora box for dozens of new issues ("hey, that rule worked, but other didn't!" or "Hey, other rule was applied, but wrongly!") that we have no capacity to support.

With that, you are more than encouraged to share it in your custom fixers repo and use it on your projects, to simplify your work.

Thanks for understanding!

**Pull request closed without merging**